pkg/cmd/util/clientcmd/clientcmd.go

```diff
@@ -68,6 +71,13 @@ func (cfg *Config) bindEnv() {
 	if value, ok := util.GetEnv("BEARER_TOKEN"); ok && len(cfg.CommonConfig.BearerToken) == 0 {
 		cfg.CommonConfig.BearerToken = value
 	}
+	if value, ok := util.GetEnv("BEARER_TOKEN_FILE"); ok && len(cfg.CommonConfig.BearerToken) == 0 {
+		if tokenData, err := ioutil.ReadFile(value); err == nil {
+			cfg.CommonConfig.BearerToken = strings.TrimSpace(string(tokenData))
+		} else {
+			glog.Error(err)
```

Comment:
Why shouldn't this method return an error and leave the handling up the caller?  It looks like a utility and the intent of the method failed.